Automattic/jetpack
# Form Block: Add a few keywords to the 'Newsletter Sign-up' variation

## Changes proposed in this Pull Request:

Adds a few relevant keywords to the 'Newsletter Sign-up' block.

From p1672677709091099-slack-C04H9D4F552

## Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

N/A

## Testing instructions:

1. Enter `/subsc` into the block editor.
2. Verify the 'Newsletter Sign-up' block appears in available options.

<img width="637" alt="image" src="https://user-images.githubusercontent.com/36432/211018292-5522511b-4df9-4a12-b518-cfae6134b213.png">
